cluster/config-defaults.yaml

```diff
@@ -1170,7 +1170,13 @@ control_plane_load_balancer_internal: "none"
 #  fs.inotify.max_user_watches = 100000
 sysctl_settings: ""
 
-
+# kube-janitor configuration
+{{if eq .Cluster.Environment "production"}}
+# This makes kube-janitor opt-in for production clusters
```

Comment:
Let's add a visible note here that before enabling this in production, one should check whether there are any currently running applications that make use of the annotation. There might be cases where people don't limit the annotation to the test environment and it propagates to production where it doesn't have an effect _so far_. Enabling it could then trigger a deletion of their application at any time.

Reply:
Hmm I agree. I'll add a note, and also double-check in the opted cluster to make sure we don't have anything  side-effects. Thanks!

Reply:
I added the note.